GoogleSignIn/Sources/GIDVerifyAccountDetail/Implementations/GIDVerifyAccountDetail.m

```diff
@@ -20,17 +20,59 @@
 #import "GoogleSignIn/Sources/Public/GoogleSignIn/GIDVerifiableAccountDetail.h"
 #import "GoogleSignIn/Sources/Public/GoogleSignIn/GIDVerifiedAccountDetailResult.h"
 
+#import "GoogleSignIn/Sources/GIDEMMSupport.h"
 #import "GoogleSignIn/Sources/GIDSignInInternalOptions.h"
 #import "GoogleSignIn/Sources/GIDSignInCallbackSchemes.h"
+#import "GoogleSignIn/Sources/GIDSignInPreferences.h"
 
-#if TARGET_OS_IOS
+@import GTMAppAuth;
 
-@implementation GIDVerifyAccountDetail
+#ifdef SWIFT_PACKAGE
+@import AppAuth;
+@import GTMSessionFetcherCore;
+#else
+#import <AppAuth/OIDAuthorizationRequest.h>
+#import <AppAuth/OIDResponseTypes.h>
+#import <AppAuth/OIDServiceConfiguration.h>
+#endif
+
+#if TARGET_OS_IOS && !TARGET_OS_MACCATALYST
+
+// The URL template for the authorization endpoint.
+static NSString *const kAuthorizationURLTemplate = @"https://%@/o/oauth2/v2/auth";
+
+// The URL template for the token endpoint.
+static NSString *const kTokenURLTemplate = @"https://%@/token";
+
+// Expected path in the URL scheme to be handled.
+static NSString *const kBrowserCallbackPath = @"/oauth2callback";
+
+// The EMM support version
+static NSString *const kEMMVersion = @"1";
+
+// Parameters for the auth and token exchange endpoints.
+static NSString *const kAudienceParameter = @"audience";
```

Comment:
Can we extract these from [`GIDSignIn`](https://github.com/google/GoogleSignIn-iOS/blob/main/GoogleSignIn/Sources/GIDSignIn.m#L127-L133) and reuse here and in `GIDSignIn` so that we don't have copy/paste them here?

Reply:
Moved them into `GIDSignIn.h`, let me know if there's a preference to move them elsewhere.

Reply:
I don't think they should be in `GIDSignIn.h` since that would expose them publicly. There is not need for that visibility in this case. Perhaps create a `GIDSignInConstants` class to encapsulate constants internally?

Reply:
Added class `GIDSignInConstants` to hold these parameters.